tests/unit/BlockSupportsAnchorTest.php

```diff
@@ -12,6 +12,9 @@ public function setUp(): void {
 		parent::setUp();
 
 		$settings                                 = get_option( 'graphql_general_settings' );
+		if ( ! is_array( $settings ) ) {
+			$settings = [];
+		}
```

Comment:
A more succint way for handling _all_ of these:

```suggestion
		$settings                                 = get_option( 'graphql_general_settings', [] );
```